sdk/python/kfp/components/_python_op.py

```diff
@@ -483,6 +483,8 @@ def get_serializer_and_register_definitions(type_name) -> str:
         '_output_files = _parsed_args.pop("_output_paths", [])',
     ])
 
+    output_serialization_code = ''.join('    ' + s + ',' + '\n' for s in output_serialization_expression_strings)
```

Comment:
nit: use '    {},\n'.format(s) to prevent the string construction from running 4 times.

Reply:
Done. Although I'm not sure it's more efficient due to the need to parse the template string and construct the new string from parts.